File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/accounting/transactions/shared/DTOs/voucher-verify.DTO.ts
export class VoucherVerify_DTO {
    public VoucherNumber: string = "";
    public Items: Array<VoucherLedgerInfo_DTO> = new Array<VoucherLedgerInfo_DTO>();
    public Remarks: string = "";
    public FiscalYearId = 0;
}

export class VoucherLedgerInfo_DTO {
    public LedgerId: number = 0;
    public Description: string = "";
    public CostCenterId = -1;
    public TransactionItemId = 0;
}

File: Code/Components/DanpheEMR.ServerModel/AccountingModels/ViewModels/VoucherVerify_DTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DanpheEMR.ServerModel.AccountingModels.ViewModels
{
    public class VoucherVerify_DTO
    {
        public string VoucherNumber { get; set; }
        public List<VoucherLedgerInfo_DTO> Items { get; set; }
        public string Remarks { get; set; }
        public int FiscalYearId { get; set; }
    }

    public class VoucherLedgerInfo_DTO
    {
        public int LedgerId { get; set; }  
        public string Description { get; set; }
        public int CostCenterId { get; set; }
        public int TransactionItemId { get; set; }
    }
}
